pgm-1.txt

```diff
@@ -0,0 +1,18 @@
+#! /bin/sh
+echo "Enter a pid:"
+read pid
+min=1
+max=32768
+if [ $pid -ge $min -a $pid -le $max ]
+	then
+		if ps -p $pid > /dev/null 2>&1
```

Comment:
Here &pid expects an input from console ,script fall over waiting for that input.The use of /dev/null is to provide empty input to program.
2>&1 is used to supress any output from stdout